lib/locator.dart

```diff
@@ -45,5 +42,7 @@ void setupLocator() {
   locator.registerLazySingleton(() => SettingsManager());
 
   // Other
+  locator.registerLazySingleton(() => SignetsAPIClient(http.Client()));
+  locator.registerLazySingleton(() => MonETSAPIClient(http.Client()));
```

Comment:
I put that here but the changes are to be applied on the package 😄 the http.Client can be optional like it was in the original SignetsApiClient. It's designed to be injected only for the tests.